app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
     pass
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(element["name"], element["age"])
+                   for element in people]
+    for person in person_list:
+        for person_dict in people:
+            if person.name == person_dict["name"]:
+                if person_dict.get("wife") is not None:
+                    person.wife = Person.people[person_dict.get("wife")]
+                if person_dict.get("husband") is not None:
+                    person.husband = Person.people[person_dict.get("husband")]
```

Comment:
Similar to the previous comment, using `Person.people.get(person_dict.get('husband'))` would prevent a potential `KeyError` if the husband's name is not found in `Person.people`.